app/services/qa/linked_data/mapper/graph_predicate_mapper_service.rb

```diff
@@ -0,0 +1,40 @@
+# Provide service for mapping predicates to object values.
+module Qa
+  module LinkedData
+    module Mapper
+      class GraphPredicateMapperService
+        class_attribute :graph_service
```

Comment:
This is just a rename of the deprecated GraphMapperService.  The code is unchanged.